src/nimvscode/nimSuggest.nim

```diff
@@ -52,7 +52,7 @@ proc provideCompletionItems*(
     return newPromise(proc (
       resolve:proc(val:seq[VscodeCompletionItem]),
       reject:proc(reason:JsObject)
-    ) = vscode.workspace.saveAll(false).then(proc() =
```

Comment:
And that fixes https://github.com/saem/vscode-nim/issues/3, I'm guessing.

Reply:
yes, for reference: https://github.com/pragmagic/vscode-nim/blob/master/src/nimSuggest.ts#L15